src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -83,7 +83,7 @@
         map(ResourceURIs.COURSE_ENROLL_STUDENTS, GET, GetCourseEnrollStudentsAction.class);
         map(ResourceURIs.INSTRUCTOR_COURSES, GET, GetInstructorCoursesAction.class);
         map(ResourceURIs.INSTRUCTOR_COURSES, POST, AddInstructorCourseAction.class);
-        map(ResourceURIs.INSTRUCTOR_COURSES_RESTORE, PUT, RestoreInstructorSoftDeletedCourseAction.class);
+        map(ResourceURIs.BIN_COURSE, DELETE, RestoreInstructorSoftDeletedCourseAction.class);
```

Comment:
- You should rename `RestoreInstructorSoftDeletedCourseAction` to `RestoreCourseAction`.
- Put the mapping around the `course` related mappings to make it more organised.

Reply:
Done!